test/backend_fused_op.in.cpp

```diff
@@ -377,3 +377,27 @@ NGRAPH_TEST(${BACKEND_NAME}, gemm_broadcast_input_C)
     test_case.add_expected_output<double>(Shape{3, 4}, vector<double>(12, 7));
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, grn_4d)
```

Comment:
Will it give pass with INTELGPU backend? If not, please add it to intelgpu's unit_test.manifest